app/views/components/ticket-panel/index.html

```diff
@@ -0,0 +1,140 @@
+{% extends "layouts/base.html" %}
+
+{% block body %}
+
+  <div class="govuk-grid-row">
+
+    <div class="govuk-grid-column-full">
+
+      {{ govukBackLink({
+        text: "Back",
+        href: "../"
+      }) }}
+
+      <h1 class="govuk-heading-xl">
+        <span class="govuk-caption-xl">Components</span> Ticket Panel
+      </h1>
+
+      {{ mojTicketPanel({
+        ariaLabel: 'Sub navigation 1',
+        items: [{
+          html: ' <h2 class="govuk-heading-m govuk-!-margin-bottom-2">This is a heading 2</h2>
+          <p class="govuk-body">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+          <a class="govuk-button govuk-!-margin-bottom-1" data-module="govuk-button">
+            Save and continue
+          </a>',
+          ariaLabel: 'Section 1'
+        }]
+      }) }}
+
+      {{ mojTicketPanel({
+        ariaLabel: 'Sub navigation 2',
+        items: [{
+          html: ' <h2 class="govuk-heading-m govuk-!-margin-bottom-2">This is a heading 2</h2>
+          <p class="govuk-body">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+          <a class="govuk-button govuk-!-margin-bottom-1" data-module="govuk-button">
+            Save and continue
+          </a>',
+          ariaLabel: 'Section 2',
+          colour: 'red'
```

Comment:
It would be more in keeping with the GOV.UK Design System to use `classes` and `attributes` to specify the additional configuration here.
```suggestion
          classes: 'moj-ticket-panel__content--red',
          attributes: {
            'aria-label': 'Section 2',
          }
```

Reply:
Have updated this to follow the same pattern, also did the parent container as well